### PR TITLE
Rolo hack

### PR DESCRIPTION
**Pull request by dsotirho-ucsc**

### CODE REVIEW

File: test/test.py [hunk 1]
```diff
@@ -46,12 +47,47 @@ def setUp(self):
...
+        donor_diseases = _get_path(test_donor, diseases_path)
+        self.assertTrue(donor_diseases == ["some_disease", "some_other_disease"])
+
+        taxon_id_path = "species.taxon_id"
+        taxon_id = _get_path(test_donor, taxon_id_path)
+        self.assertTrue(taxon_id == "9606")
+
+
+    def test_ontology_label(self):
+        ontology = {'text': 'foo', 'ontology': 'bar', 'ontology_label': 'baz',}
```
Comment (hannes-ucsc) [inline test/test.py]: `ontology_label` already has doctests and those are run by the Travis build. Please move any cases from here that  are not covered in the doctest to the doctest and remove this test.

**Review by hannes-ucsc — CHANGES_REQUESTED** [review]
This seems a little too immature for a review. There are substantial amounts of commented-out code, conflicts, duplicate tests and the tests aren't even passing. Was this supposed to be a request for a preliminary review as a WIP?

I think the above issues should be addressed before another review is requested, even a preliminary one. If could stomach reviewing this as a WIP with the issues (conflicts,  commented out code, ...) unresolved IF I am being asked to look at a specific area of the code.

**Comment by hannes-ucsc** [comment]
Ah, the tests *are* passing. Let me take another stab at the review.

**Comment by simonjupp** [comment]
We definitely rushed and hacked some bits to get something functional. I figured you would want to do some refactoring to integrate this properly. 😀

File: src/humancellatlas/data/metadata/lookup.py [hunk 2]
```diff
@@ -18,48 +110,115 @@ def lookup(d: Mapping[K, V], k: K, *ks: K, default: Union[V, LookupDefault] = Lo
...
     """
+
+    if 'schema_version' in d or 'describedBy' in d:
+        version = get_document_version(d)
+        new_k = schema_template.replaced_by_at(k, version)
+    else:
+        version = '0.0.0'
+        new_k = k
+
+    # HACK for missing entries in https://schema.dev.data.humancellatlas.org/property_migrations
```
Comment (hannes-ucsc) [inline src/humancellatlas/data/metadata/lookup.py]: @simonjupp, do you happen to know if these migrations were added?

Comment (simonjupp) [reply src/humancellatlas/data/metadata/lookup.py]: No, I think we only have migrations from when the migration file was introduced. Do you think it’s worth retrofitting now? If these are really old I’d say no, if they are needed for data currently in production then we can probably put them in.

**Comment by hannes-ucsc** [comment]
Closing prior to archival of repository.

**Pull request closed without merging**